libs/langchain-google-genai/src/chat_models.ts

```diff
@@ -244,6 +247,8 @@ export class ChatGoogleGenerativeAI
 
   streamUsage = true;
 
+  json = false
```

Comment:
If this is only accessed inside the class constructor we don't need to make it a class property.
```suggestion
```